spec/server/server.test.js

```diff
@@ -87,6 +92,89 @@ describe('Event API', () => {
       });
     });
   });
+
+  describe('Integration tests', () => {
+    const url = `${server}/event/${MAX_NUMBER_OF_EVENTS}`;
+
+    let sample = {
+      title: 'Testing API integration',
+      local_date_time: new Date(),
+      orgId: 'o10',
+      series: {
+        description: 'Every 2nd Sunday of the month until April 2020',
+        frequency: {
+          day_of_week: 'Monday',
+          interval: 2,
+        },
+      },
+    };
+    let options = {
+      url,
+      body: sample,
+      json: true
+    }
+
+    test('GET, POST, PUT, DELETE on /event/:eventId & GET on /event/timedate/:eventId', (done) => {
+
```

Comment:
When this test fails, do you know where to go to find the error? It looks like it'll point you to a file not a method